src/core/github/client.py

```
@@ -0,0 +1,46 @@
+import subprocess
+from collections.abc import Mapping
+
+
+def run_cmd(
+    cmd: list[str],
+    *,
+    capture_output: bool = True,
+    env: Mapping[str, str] | None = None,
+) -> subprocess.CompletedProcess:
+    """Run *cmd* as a subprocess and return the completed process."""
+    return subprocess.run(cmd, check=False, capture_output=capture_output, text=True, env=env)
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)